thirdeye-ui/src/app/components/alert-wizard-v2/alert-details/alert-frequency/alert-frequency.component.tsx

```diff
@@ -46,10 +46,7 @@ function AlertFrequency<NewOrExistingAlert extends EditableAlert | Alert>({
                 </Box>
             </Grid>
 
-            <AlertDateTimeCronAdvance
-                cron={currentCron}
-                onCronChange={handleCronChange}
-            />
+            <CronEditor value={currentCron} onChange={handleCronChange} />
         </>
```

Comment:
The switch between `Date/Time` and `Cron` should in this component and we should continue using `AlertDateTimeCronAdvance` for the `Cron` mode